.lighthouserc-1.js

```diff
@@ -88,10 +88,9 @@ module.exports = {
             // The YouTube embed on donate page uses passive listeners.
             'uses-passive-event-listeners': ['error', {minScore: 0}],
             'uses-rel-preload': ['error', {minScore: 1}],
-            deprecations: [
-              'error',
-              {minScore: 1, ignoredPatterns: ['window.webkitStorageInfo']},
-            ],
+            // TODO(#20286): There is a depdeprecated API on the /donate page due to the
```

Comment:
@jnvtnguyen This should be "deprecated", right?

Maybe better to fix in a follow-up PR -- I'm merging this since it is blocking folks. Thanks for making the fix!

Reply:
(Though if you see this before the merge is done, feel free to just push the fix to this PR too :)  Thanks!)